### PR TITLE
Enable custom NodePorts for service

### PR DESCRIPTION
**Pull request by jonmosco**
When the serviceType is set to NodePort, the chart will pick the port randomly (from the API servers specified port range).   

The ability to set these ports specifically is needed sometimes when running this in production.  

```yaml
ServiceType: NodePort
nodePorts:
    http: 30001
    https: 30002
```

This was tested and linted to follow the CONTRIBUTION guide.

### CODE REVIEW

File: stable/wordpress/templates/svc.yaml [hunk 1]
```diff
@@ -10,11 +10,17 @@ metadata:
...
-    port: 80
-    targetPort: http
-  - name: https
-    port: 443
-    targetPort: https
+    - name: http
+      port: 80
+      targetPort: http
+      {{- if (and (eq .Values.serviceType "NodePort") (not (empty .Values.nodePort.http)))}}
+      nodePort: {{ default "" .Values.nodePort.http }}
```
Comment (eicnix) [inline stable/wordpress/templates/svc.yaml]: Why do you provide a default parameter if you check in the line before that `nodePort.http` is set?

Comment (jonmosco) [reply stable/wordpress/templates/svc.yaml]: Yes, good point.  I corrected this and retested for lint checks.  Thank you for pointing that out.

**Comment by javsalgar** [comment]
I checked it and LGTM. However I am not sure if an explicit NodePort is the best practice (I assume it is an alternative when you do not have a LoadBalancer).

File: stable/wordpress/values.yaml [hunk 2]
```diff
@@ -97,6 +97,14 @@ mariadb:
 ## For minikube, set this to NodePort, elsewhere use LoadBalancer
 ##
 serviceType: LoadBalancer
+##
+## serviceType: NodePort
+## nodePort:
```
Comment (prydonius) [inline stable/wordpress/values.yaml]: nodePort**s**

File: stable/wordpress/README.md [hunk 3]
```diff
@@ -67,7 +67,10 @@ The following tables lists the configurable parameters of the WordPress chart an
 | `mariadb.mariadbUser`                | Database user to create                    | `bn_wordpress`                                 |
 | `mariadb.mariadbPassword`            | Password for the database                  | _random 10 character long alphanumeric string_ |
 | `serviceType`                        | Kubernetes Service type                    | `LoadBalancer`                                             |
-| `healthcheckHttps`                   | Use https for liveliness and readiness     | `false`                                             |
+| `NodePort`                           | Kubernetes node port                       | Not set                                                    |
```
Comment (prydonius) [inline stable/wordpress/README.md]: I'm not sure what this line is for, it's not an actual value you can set?

**Comment by prydonius** [comment]
/ok-to-test

@jonmosco could you rebase this?

**Review by sameersbn — APPROVED** [review]

**Comment by sameersbn** [comment]
thanks for the review @javsalgar @prydonius 
/lgtm

**Pull request merged**